openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -37,7 +39,7 @@ p {
 }
 
 pre {
-  font-family: 'Inter', sans-serif;
+  font-family: 'Poppins', sans-serif;
```

Comment:
For code block we need to update the fonts right?

Reply:
Updated